enip_backend/export/schemas/national.example.json

```diff
@@ -0,0 +1,443 @@
+{
+  "national_summary": {
+    "P": [
+      {
+        "name": "Clinton",
+        "party": "Dem",
+        "winner": false,
+        "popVote": 65853516,
+        "popPct": 0.49,
+        "statesWon": 21,
+        "electWon": 232
+      },
+      {
+        "name": "Trump",
+        "party": "GOP",
+        "winner": true,
+        "popVote": 62984825,
+        "popPct": 0.46,
+        "statesWon": 30,
+        "electWon": 306
+      },
+      {
+        "name": "Johnson",
+        "party": "Lib",
+        "winner": false,
+        "popVote": 4489221,
+        "popPct": 0.03,
+        "statesWon": 0,
+        "electWon": 0
+      }
+    ],
+    "S": [
+      {
+        "won": 22,
```

Comment:
I think we'll want to put in a key with the number of holdover seats

Reply:
going to just hardcode this in the frontend

Reply:
front-end can deal with holdover math